crates/sncast/src/starknet_commands/script.rs

```diff
@@ -51,131 +53,24 @@ pub struct Script {
     pub script_module_name: String,
 }
 
-pub struct CairoHintProcessor<'a> {
+pub struct CastExtension<'a> {
```

Comment:
I think moving `CastExtension` to another file and just implementing `run` is this one would be good, but prob out of scope of this PR

Reply:
It is probably CastScriptExtension

Reply:
renamed